worlds/tunic/__init__.py

```diff
@@ -250,7 +250,7 @@ def extend_hint_information(self, hint_data: Dict[int, Dict[int, str]]):
                     name, connection = connection
                     # for LS entrances, we just want to give the portal name
                     if "(LS)" in name:
-                        name, _ = name.split(" (LS) ")
+                        name = name.split(" (LS) ")[0]
```

Comment:
```suggestion
                        name = name.split(" (LS) ", 1)[0]
```
if we're optimizing anyway, may as well go all the way.